contracts/NamefiServiceCredit.sol

```diff
@@ -165,6 +188,20 @@ contract NamefiServiceCredit is
         return keccak256("SUCCESS");
     }
 
+    /**
+     * @notice Sets the Chainlink price feed address
+     * @param priceFeedAddress The address of the ETH/USD price feed
+     */
+    function setEthUsdPriceFeed(address priceFeedAddress) external onlyRole(DEFAULT_ADMIN_ROLE) {
```

Comment:
I am curious, why do we need to input `priceFeedAddress` each time? Do you envision it will change?